lib/stream-mmmagic.js

```diff
@@ -1,17 +1,41 @@
-var mmm = require('mmmagic');
-var peek = require('buffer-peek-stream');
+var mmm = require('mmmagic'),
+    peek = require('buffer-peek-stream'),
+    _ = require('lodash');
 
-function streamMmmagic(stream, callback) {
+function streamMmmagic(stream, optionsOpt, callback) {
+  var args = [], 
+    _options,
+    i;
+  for (i = 0; i < arguments.length; i++) {
+    args.push(arguments[i]); 
+  }
+  stream = args.shift();
+  callback = args.pop();
+  if (args.length > 0 && typeof args[0] === 'object') {
+    optionsOpt = args[0];
+  } else {
+    optionsOpt = {};
+  }
+  _options = _.merge({
+    splitMime: true
+  }, config, optionsOpt);
   // peek 16K which is more than you need for type but gives libmagic a better chance of
   // accurately detecting the encoding
   return peek(stream, 16384, function (err, buf, dest) {
     if (err) return callback(err, null, dest);
-    var magic = config.magicFile
-        ? new mmm.Magic(config.magicFile, mmm.MAGIC_MIME)
-        : new mmm.Magic(mmm.MAGIC_MIME);
+    var magic;
+    if (_options.magicFile) {
+      magic = new mmm.Magic(config.magicFile, mmm.MAGIC_MIME);
```

Comment:
looks like this is broken; `_options.magicFile` is the option, but `config.magicFile` is what's sent.  Switching them would break backwards compatibility since you have 3 call options here; call with options, call with config, call without magic file.